src/elements/forms/text-field.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --component-padding: 1rem;
+  --label-padding-x: .25em;
+  --label-padding-y: .125em;
+  --form-control-corner-radius: var(--component-corner-radius, 2px);
+  --form-control-padding-x: var(--component-padding, 1rem);
+  --form-control-padding-y: var(--component-padding, 1rem);
+}
+
+/* Global styles */
+
+/* TODO: Move to Foundation partially */
+
+input {
+  display: block;
+  width: 100%;
+  padding: var(--form-control-padding-y) var(--form-control-padding-x);
+  border: 1px solid var(--primary);
+  border-radius: var(--form-control-corner-radius, 2px);
+  height: calc(2 * var(--form-control-padding-y) + 1em);
+
+  &:focus {
```

Comment:
Didn't you want to avoid this syntax?

Reply:
No, this particular syntax is useful. I wanted to avoid nesting that constructs additional classes such as `&__child` because it might confuse and makes searching tricky. In this case it's clear.